api-client/src/endpoints/hooks/useLoadFromEndpoint.ts

```diff
@@ -39,7 +39,7 @@ type ReturnType<T> = {
 export const useLoadFromEndpoint = <T>(request: RequestType<T>): ReturnType<T> => {
   const [data, setData] = useState<T | null>(null)
   const [error, setError] = useState<Error | null>(null)
-  const [loading, setLoading] = useState<boolean>(false)
+  const [loading, setLoading] = useState<boolean>(true)
```

Comment:
Initially this function will return `{null, false, null}`, probably as it is called before the first render, I think. To not show an error (before the first redirect) we can initially set the state to `loading = true` as each endpoint handles this state correctly